mediaqueries/Overview.bs

```diff
@@ -1447,6 +1447,49 @@ inverted-colors</h3>
 		</pre>
 	</div>
 
+<h3 id="color-gamut">
+color-gamut</h3>
+
+	<pre class='descdef mq'>
+	Name: color-gamut
+	Value: srgb | p3 | rec2020
+	For: @media
+	Type: range
+	</pre>
+
+	The 'color-gamut' media feature describes the approximate range of colors that are
+	supported by the browser and display device. That is, if the browser receives content
+	with colors in the specified space it can cause the display to render the appropriate
+	color, or something appropriately close enough.
+
+	The query uses approximate ranges for a few reasons. Firstly, there are a lot of differences in
```

Comment:
p3 or dci_p3, slightly more descriptive (but longer)?
if shorter is preferred, maybe change rec2020 to 2020?